openupgrade_framework/odoo_patch/odoo/models.py

```diff
@@ -40,3 +40,36 @@ def unlink(self):
 
 unlink._original_method = BaseModel.unlink
 BaseModel.unlink = unlink
+
+
+def _load_records(self, data_list, update=False):
+    for vals in data_list:
+        xml_id = vals.get("xml_id", False)
+        original_record = self.env.ref(xml_id, raise_if_not_found=False)
+        arch_db = vals.get("values").get("arch", False)
```

Comment:
@duong77476 thêm như này cho chắc
```suggestion
        arch_db = (vals.get("values") or {}).get("arch", False)
```

Reply:
Vaang a